content/kotlin/concepts/hashmap/terms/get.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.get()'
+Description: 'The get function is used to retrieve the value associated with a particular key in a HashMap.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'HashMap'
+  - 'Data Types'
+  - 'Collections'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+In Kotlin, the `get()` function is a method of the Map interface that is used to retrieve the value associated with a particular key.
+
+## Syntax
+
+```pseudo
+fun <K, V> Map<K, V>.get(key: K): V?
+```
+
+Where,`Map<K, V>` is the interface and the `get()` function is the method of this interface.`K` is the type of the key of the map. `V` is the type of the value associated with the key.`key` is the key for which the associated value should be retrieved.
```

Comment:
```suggestion
The `key` parameter is of [data type](https://www.codecademy.com/resources/docs/kotlin/data-types) `K`. If the `key` exists in `myHashMap`, the associated value, of type `V`, is returned. Otherwise, `null` is returned.
```